application/common/models/Verification.php

```diff
@@ -0,0 +1,113 @@
+<?php
+namespace common\models;
+
+use common\models\User;
+use Sil\IdpPw\Common\PhoneVerification\NotMatchException;
+use yii\base\Model;
+
+class Verification extends Model
+{
+    const TYPE_EMAIL = 'email';
+    const TYPE_PHONE = 'phone';
+
+    /**
+     * Send email for verification
+     * @param string $toAddress
+     * @param string $subject
+     * @param string $view Full view path with alias, ex: @common/mail/reset/self
+     * @param string $code
+     * @param User $forUser
+     * @param null|string $ccAddress
+     * @param null|integer $eventLogUserId
+     * @param null|string $eventLogTopic
+     * @param null|string $eventLogDetails
+     */
+    public static function sendEmail(
+        $toAddress,
+        $subject,
+        $view,
+        $code,
+        $forUser,
+        $ccAddress = null,
+        $eventLogUserId = null,
+        $eventLogTopic = null,
+        $eventLogDetails = null
+    ) {
+        $body = \Yii::$app->mailer->render(
+            $view,
+            [
+                'idpName' => \Yii::$app->params['idpName'],
+                'name' => $forUser->first_name,
+                'code' => $code,
+                'toAddress' => $toAddress,
+            ]
+        );
+
+        EmailQueue::sendOrQueue(
+            $toAddress,
+            $subject,
+            $body,
+            $body,
```

Comment:
My expectation is it just would display all the html as text if their client only supported text emails, which I'm okay with for now because I'm not aware of any clients like that that would be in use by our folks.